pkg/sshd/server_test.go

```diff
@@ -8,6 +10,7 @@ import (
 	"github.com/deis/builder/pkg/controller"
 
 	"github.com/Masterminds/cookoo"
+	"github.com/arschles/assert"
```

Comment:
>dont want to be a stickler here but should this be in pkg

I don't think so - we could fork it to the Deis org if you like. I've used this elsewhere as-is though.

>can we use antoher matching lib like gomega?

I'd prefer not to in this particular case because Gomega is a larger dependency that includes a lot of functionality not needed here.